hudi-client/hudi-client-common/src/test/java/org/apache/hudi/common/table/timeline/TestHoodieArchivedTimeline.java

```diff
@@ -83,7 +84,7 @@ private void writeArchivedTimeline(int batchSize, long startTs) throws Exception
         .withIndexConfig(HoodieIndexConfig.newBuilder().withIndexType(HoodieIndex.IndexType.INMEMORY).build())
         .withMarkersType("DIRECT")
         .build();
-    HoodieEngineContext engineContext = new HoodieLocalEngineContext(new Configuration());
+    HoodieEngineContext engineContext = new HoodieLocalEngineContext(HadoopFSUtils.getStorageConf(new Configuration()));
```

Comment:
Can we use `HoodieTestUtils.getDefaultStorageConf()` ?

Reply:
Fixed and `HoodieTestUtils.getDefaultStorageConf()` is used in all proper places.